big_tests/tests/push_http_SUITE.erl

```diff
@@ -153,15 +153,17 @@ push_to_many(Config) ->
         end).
 
 start_pool() ->
-    ejabberd_node_utils:call_fun(mongoose_http_client, start, [[]]),
-    ejabberd_node_utils:call_fun(mongoose_http_client,
-        start_pool,
-        [http_pool, [{server, "http://localhost:8000"}]]),
+    Pool = {http, host, http_pool,
+            [{strategy, random_worker}, {call_timeout, 5000}, {workers, 20}],
+            [{path_prefix, "/"}, {http_opts, []}, {server, "http://localhost:8000"}]},
+    ejabberd_node_utils:call_fun(mongoose_wpool,
+                                 start_configured_pools,
+                                 [[Pool], [<<"localhost">>]]),
+    ejabberd_node_utils:call_fun(mongoose_wpool_http, init, []),
```

Comment:
This line is not needed. The pool will be initialized by `start_configured_pools` function here: https://github.com/esl/MongooseIM/blob/943ebab61e654d9301414615416e883de5fbf7fb/src/wpool/mongoose_wpool.erl#L87